cmake/Config.cmake.in

```diff
@@ -10,9 +10,23 @@ set(@PROJECT_NAME@_VERSION @PROJECT_VERSION@)
 # find_dependency(Imath @Imath_VERSION@ HINTS @Imath_DIR@)
 
 set_and_check (@PROJECT_NAME@_DIR "${PACKAGE_PREFIX_DIR}")
+if (WIN32)
+    # For .dll files.
+    set_and_check (@PROJECT_NAME@_BINARY_DIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_BINDIR@")
```

Comment:
Is this only needed for shared builds for the DLLs (going by the comment)?

Is it needed to be set for static builds?
(Just thinking that it checks that the folder exists, but it might be empty in that case)

Reply:
Oh good point, the reason for the `if (WIN32)` was exactly that - `set_and_check` errored on Linux because there are no binaries.

Reply:
Added `BUILD_SHARED_LIBS` to the conditional in d66313bc